test/apify_api_error.test.js

```diff
@@ -62,4 +63,56 @@ describe('ApifyApiError', () => {
         expect(error.httpMethod).toEqual('get');
         expect(error.attempt).toEqual(1);
     });
+
+    test('should carry additional error data if provided', async () => {
+        const server = await mockServer.start();
```

Comment:
Don't forget to `close()` what you open :) 

Btw in all the other tests, we're starting the server in `beforeAll()` (and closing it in `afterAll()`). I know that here it's only for this one test, but for the sake of consistency, can we move it? It's just a matter of time before we want to add a new test calling the mock server.

Reply:
Yea I was not sure what would be the correct approach here since I do think there will be other tests here in the future. But there is no harm in moving it there so I will do it, at least it will be consistent.